osu.Game.Rulesets.Osu/Difficulty/Skills/Flashlight.cs

```diff
@@ -84,6 +87,19 @@ private double strainValueOf(DifficultyHitObject current)
             if (hidden)
                 result *= 1.0 + hidden_bonus;
 
+            double sliderBonus = 0.0;
+
+            if (osuCurrent.TravelTime != 0)
+            {
+                // Reward sliders based on velocity.
+                double normalisedTravelDistance = osuCurrent.TravelDistance / scalingFactor;
```

Comment:
Unless I'm missing something, `TravelDistance` is already being normalized in ODHO - https://github.com/ppy/osu/blob/7d93778355f7edb8d4c674f0707d69bcf5647c4e/osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs#L213

Reply:
This was actually intentional as it's de-normalising the distance. I've refactored + documented better.